libtextsecure/account_manager.js

```diff
@@ -532,12 +532,38 @@
         'private'
       );
 
+      /*
+      // Ensure that we always have a group conversation for comparisong
+      const debugConversation = await ConversationController.getOrCreateAndWait(
+        'debug',
+        'group'
+      );
+      await debugConversation.setFriendRequestStatus(4);
+      await debugConversation.set(updates);
+      */
+      const updates = {
+        name: 'GroupChat',
+        active_at: Date.now(),
+      };
+
+      // Ensure that we always have a public chat conversation for ourself
+      const publicChatConversation = await ConversationController.getOrCreateAndWait(
+        'lokiPublicChat',
+        'public'
+      );
+      updates.name = 'PublicChat';
+      await publicChatConversation.setFriendRequestStatus(4);
+      await publicChatConversation.set(updates);
+
       await storage.setProfileName(profileName);
 
       // Update the conversation if we have it
       const newProfile = storage.getLocalProfile();
       await conversation.setProfile(newProfile);
 
+      // await debugConversation.setProfile(newProfile);
+      await publicChatConversation.setProfile(newProfile);
```

Comment:
Isn't this setting the name of the public chat to your own personal profile name?

Reply:
Thought it was setting owner information on the conversation (though not really sure) because it's not changing the name to personal profile name.